diff --git a/spa/plugins/alsa/mixer/paths/analog-input-internal-mic-always.conf b/spa/plugins/alsa/mixer/paths/analog-input-internal-mic-always.conf
index 9e22008c9..ac43525f9 100644
--- a/spa/plugins/alsa/mixer/paths/analog-input-internal-mic-always.conf
+++ b/spa/plugins/alsa/mixer/paths/analog-input-internal-mic-always.conf
@@ -19,7 +19,7 @@
 ; See analog-output.conf.common for an explanation on the directives
 
 [General]
-priority = 89
+priority = 79
 description-key = analog-input-microphone-internal
 
 [Jack Mic]
diff --git a/spa/plugins/alsa/mixer/paths/analog-input-internal-mic.conf b/spa/plugins/alsa/mixer/paths/analog-input-internal-mic.conf
index 898410acc..bcec82017 100644
--- a/spa/plugins/alsa/mixer/paths/analog-input-internal-mic.conf
+++ b/spa/plugins/alsa/mixer/paths/analog-input-internal-mic.conf
@@ -19,7 +19,7 @@
 ; See analog-output.conf.common for an explanation on the directives
 
 [General]
-priority = 89
+priority = 79
 description-key = analog-input-microphone-internal
 
 [Jack Mic]
diff --git a/spa/plugins/alsa/mixer/paths/analog-output-speaker-always.conf b/spa/plugins/alsa/mixer/paths/analog-output-speaker-always.conf
index 756afa954..ec864eb8c 100644
--- a/spa/plugins/alsa/mixer/paths/analog-output-speaker-always.conf
+++ b/spa/plugins/alsa/mixer/paths/analog-output-speaker-always.conf
@@ -19,7 +19,7 @@
 ; See analog-output.conf.common for an explanation on the directives
 
 [General]
-priority = 100
+priority = 80
 description-key = analog-output-speaker
 
 [Properties]
diff --git a/spa/plugins/alsa/mixer/paths/analog-output-speaker.conf b/spa/plugins/alsa/mixer/paths/analog-output-speaker.conf
index 72f928fd9..7d2f8cf53 100644
--- a/spa/plugins/alsa/mixer/paths/analog-output-speaker.conf
+++ b/spa/plugins/alsa/mixer/paths/analog-output-speaker.conf
@@ -18,7 +18,7 @@
 ; See analog-output.conf.common for an explanation on the directives
 
 [General]
-priority = 100
+priority = 80
 description-key = analog-output-speaker
 
 [Properties]
