diff --git a/src/sdlwindow.cpp b/src/sdlwindow.cpp
index 8986e02..ca616d6 100644
--- a/src/sdlwindow.cpp
+++ b/src/sdlwindow.cpp
@@ -134,7 +134,10 @@ void inputSDLThreadRun( void )
 			case SDL_KEYUP:
 				key = SDLScancodeToLinuxKey( event.key.keysym.scancode );
 
-				if ( event.type == SDL_KEYUP && ( event.key.keysym.mod & KMOD_LGUI ) )
+				if ( key == KEY_RIGHTMETA || key == KEY_LEFTMETA )
+					break;
+
+				if ( event.type == SDL_KEYUP && ( event.key.keysym.mod & KMOD_RALT ) )
 				{
 					bool handled = true;
 					switch ( key )
