diff --git a/src/sdlwindow.cpp b/src/sdlwindow.cpp
index ca616d6..cb09f82 100644
--- a/src/sdlwindow.cpp
+++ b/src/sdlwindow.cpp
@@ -44,8 +44,8 @@ static inline int SDLButtonToLinuxButton( int SDLButton )
 		case SDL_BUTTON_LEFT: return BTN_LEFT;
 		case SDL_BUTTON_MIDDLE: return BTN_MIDDLE;
 		case SDL_BUTTON_RIGHT: return BTN_RIGHT;
-		case SDL_BUTTON_X1: return BTN_FORWARD;
-		case SDL_BUTTON_X2: return BTN_BACK;
+		case SDL_BUTTON_X1: return BTN_EXTRA;
+		case SDL_BUTTON_X2: return BTN_SIDE;
 		default: return 0;
 	}
 }
diff --git a/src/wlserver.cpp b/src/wlserver.cpp
index d405b47..dd591f3 100644
--- a/src/wlserver.cpp
+++ b/src/wlserver.cpp
@@ -229,6 +229,10 @@ static inline uint32_t steamcompmgr_button_to_wlserver_button( int button )
 			return BTN_RIGHT;
 		case 3:
 			return BTN_MIDDLE;
+		case 4:
+			return BTN_EXTRA;
+		case 5:
+			return BTN_SIDE;
 	}
 }
 
diff --git a/src/wlserver.hpp b/src/wlserver.hpp
index 9928636..b7bc84d 100644
--- a/src/wlserver.hpp
+++ b/src/wlserver.hpp
@@ -5,7 +5,7 @@
 #include <wayland-server-core.h>
 #include <atomic>
 
-#define WLSERVER_BUTTON_COUNT 4
+#define WLSERVER_BUTTON_COUNT 5
 #define WLSERVER_TOUCH_COUNT 11 // Ten fingers + nose ought to be enough for anyone
 
 struct wlserver_t {
