diff --git a/proton b/proton
index 1c7a9e1..d2881bd 100755
--- a/proton
+++ b/proton
@@ -1558,11 +1558,7 @@ class Session:
         else:
             remote_debug_proc = None
 
-        # CoD: Black Ops 3 workaround
-        if os.environ.get("SteamGameId", 0) == "311210":
-            rc = self.run_proc([g_proton.wine_bin, "c:\\Program Files (x86)\\Steam\\steam.exe"] + sys.argv[2:] + self.cmdlineappend)
-        else:
-            rc = self.run_proc([g_proton.wine64_bin, "c:\\windows\\system32\\steam.exe"] + sys.argv[2:] + self.cmdlineappend)
+        rc = self.run_proc([g_proton.wine64_bin] + sys.argv[2:] + self.cmdlineappend)
 
         if remote_debug_proc:
             remote_debug_proc.kill()
