diff --git a/config.c b/config.c
index ff7e7db..95d4a04 100644
--- a/config.c
+++ b/config.c
@@ -716,6 +716,7 @@ static char *get_config_path(void) {
 	char *config_paths[] = {
 		mako_asprintf("%s/.mako/config", home),
 		mako_asprintf("%s/mako/config", config_home),
+		mako_asprintf("%s/mako/config", "/etc/xdg"),
 	};
 
 	size_t config_paths_len = sizeof(config_paths) / sizeof(config_paths[0]);
