diff --git a/setup.py b/setup.py
index 196c53f..fa8c47d 100644
--- a/setup.py
+++ b/setup.py
@@ -29,11 +29,13 @@ OPENSUSE = ARCH_LINUX + [
 FEDORA = ARCH_LINUX + [
     "harfbuzz",
 ]
+NIXOS = OPENSUSE
 LIBRARIES = {
     "default": DEFAULT,
     "ubuntu": DEFAULT,
     "arch": ARCH_LINUX,
     "manjaro": ARCH_LINUX,
+    "nixos": NIXOS,
     "artix": ARCH_LINUX,
     "opensuse": OPENSUSE,
     "fedora": FEDORA,
