diff --git a/internal/configuration/decode_hooks.go b/internal/configuration/decode_hooks.go
index a5872627..87ae6a0a 100644
--- a/internal/configuration/decode_hooks.go
+++ b/internal/configuration/decode_hooks.go
@@ -513,6 +513,13 @@ func StringToPasswordDigestHookFunc(plaintext bool) mapstructure.DecodeHookFuncT
 
 		var result *schema.PasswordDigest
 
+		if strings.HasPrefix(dataStr, "$file$") {
+			path := strings.TrimPrefix(dataStr, "$file$")
+			if dataStr, err = loadSecret(path); err != nil {
+				return nil, fmt.Errorf(errFmtSecretIOIssue, path, expectedType.String(), err);
+			}
+		}
+
 		if !strings.HasPrefix(dataStr, "$") {
 			dataStr = fmt.Sprintf(crypt.StorageFormatSimple, crypt.AlgorithmPrefixPlainText, dataStr)
 		}
